ExampleMod/Content/Tiles/Plants/ExampleTree.cs

```diff
@@ -41,12 +45,12 @@ public class ExampleTree : ModTree
 
 		// Branch Textures
 		public override Asset<Texture2D> GetBranchTextures() {
-			return ModContent.Request<Texture2D>("ExampleMod/Content/Tiles/Plants/ExampleTree_Branches");
+			return branchesTexture ??= ModContent.Request<Texture2D>("ExampleMod/Content/Tiles/Plants/ExampleTree_Branches");
```

Comment:
Do we want to show using `Mod.Assets.Request` here? Or are we happy leaving that for 'advanced' modders and sticking to `ModContent` with full paths?

Reply:
The plant classes don't actually have a Mod field, it seems.